doc/Makefile

```diff
@@ -187,4 +187,4 @@ uml:
 .PHONY: apidoc
 apidoc:
 	-rm -rf source/api/
-	sphinx-apidoc -o source/api ../enthought_example
+	sphinx-apidoc -H 'Enthought plugins' -M -o source/api ../ ../setup.py ../ci ../*tests*
```

Comment:
Definitely long overdue removing the tests from documentation! Also i'm surprised you have to give sphinx a flag to make it put the module documentation first, seemed like that would be a sensible default